multiview_mapping_toolkit/cameras/cameras.py

```diff
@@ -731,6 +732,14 @@ def vis(
         if plotter is None:
             plotter = pv.Plotter()
             show = True
+        
+        # Determine pairwise distance between each camera and set frustum_scale to 1/120th of the maximum distance found
+        max_distance = 0
+        if frustum_scale is None and self.n_cameras() >= 2:
+            camera_translation_matrices = np.array([transform[:3, 3] for transform in self.cam_to_world_transforms])
+            distances = pdist(camera_translation_matrices, metric='euclidean')
+            max_distance = np.max(distances)
+            frustum_scale = max_distance / 120 if max_distance > 0 else 1
```

Comment:
I assume this is in case all cameras are at the same location? That's smart, I wouldn't have thought of that. 

Can you add parentheses to show the `max_distance / 120` is the term on the left side of the conditional and this isn't choosing the denominator of the fraction? If `black` reverts the change, no worries, just maybe leave a comment.